app/views/idv/sessions/step1.html.slim

```diff
@@ -0,0 +1,48 @@
+- title t('idv.titles.welcome')
```

Comment:
well, it'd actually need to be `show.html.slim`, wouldn't it? (With which I'm ok, but it doesn't match the method name)